package.json

```diff
@@ -63,5 +63,6 @@
   "volta": {
     "node": "18.6.0",
     "pnpm": "8.5.1"
-  }
+  },
+  "packageManager": "yarn@1.22.17"
```

Comment:
????

Reply:
If I don't define the yarn version, `postinstall` command for `cardpay-sdk` depedency will be failed because it executes `yarn codegen`. I don't know if that issue only in my local or not.

Reply:
Nvm, I can just change the default version of yarn on my local computer without defining it in the project.

Reply:
ill make a ticket to look into this.